shuup/core/signals.py

```diff
@@ -19,7 +19,7 @@
 get_basket_command_handler = Signal(providing_args=["command"], use_caching=True)
 pre_clean = Signal(providing_args=["instance"], use_caching=True)
 post_clean = Signal(providing_args=["instance"], use_caching=True)
-context_cache_item_bumped = Signal(providing_args=["item"], use_caching=True)
+context_cache_item_bumped = Signal(providing_args=["shop_id"], use_caching=True)
```

Comment:
well, this is certainly not backwards compatible, right? I mean, some 3rd party apps can use this to bump caches passing the item, but in the signal handler here https://github.com/shuup/shuup/pull/2325/files#diff-2dfff9a8fb51a9f6302cb45d759c752ab0ffd8583cfeab9fed9b35652f83b4d1R23 you don't check for that key, you probably should do that.